cmd/keycloakb/keycloak_bridge.go

```diff
@@ -530,6 +530,7 @@ func main() {
 	healthChecker.AddDatabase("Users R/W", usersRwDBConn, healthCheckCacheDuration)
 	healthChecker.AddDatabase("Archive RO", archiveRwDBConn, healthCheckCacheDuration)
 	healthChecker.AddHTTPEndpoint("Keycloak", keycloakConfig.AddrAPI, httpTimeout, 200, healthCheckCacheDuration)
+	healthChecker.AddHTTPEndpoint("Accounting", c.GetString(cfgAddrAccounting), httpTimeout, 200, healthCheckCacheDuration)
```

Comment:
I had in mind you have reverted it to avoid some issues

Reply:
Of course! PR was not correctly rebased